### PR TITLE
Todo list structure

### PR DESCRIPTION
**Pull request by NatanimA**
> Inserted Todo list project structure

> Followed Html and CSS best practices

> Inserted webpack and loaders to the project

> Fetching of data dynamically by Javascript objects

`Please do this to run it on your pc`

`npm start` =>this will prompt you to install webpack then `y` and it will install

`npm run build` => this will generate codes to the dist folder

`npm start` and this will start the server and you can see the the project I did 😉

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -20,9 +20,18 @@
   },
   "homepage": "https://github.com/NatanimA/To-Do-List-App#readme",
   "devDependencies": {
```
Comment (GzimAsani) [inline package.json:22]: - [ ] I should be able to run your project with the start command, kindly check your scripts and webpack 
image

**Review by GzimAsani — CHANGES_REQUESTED** [review]
### Hello @NatanimA  :wave:   
 Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!
 
 ## Highlights
 * Good job with your project  :+1:  
 
 ## Required Changes 

- [ ] Check the comment below :arrow_double_down: 






 ## Optional suggestions
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding! clapclapclap
 
 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
 
 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: package.json (hunk 1 above)
Comment (NatanimA) [reply package.json:22]: It is because I added node_modules in gitignore, should I remove it and push that as well? @GzimAsani

Comment (GzimAsani) [reply package.json:22]: No, node_modules should be in gitignore

Comment (NatanimA) [reply package.json:22]: Then why is it not working?, It is working on mine? @GzimAsani ,Please any suggestions

Comment (NatanimA) [reply package.json:22]: > No, node_modules should be in gitignore

> It is working on mine you see. @GzimAsani 

Capture

**Review by Davidosky007 — APPROVED** [review]
Hi @NatanimA 👋 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
* No linter errors 👍 
* Well written PR description 👍 
* Good looking readme file 👍  
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**